aws/static-site-s3/s3.tf

```diff
@@ -1,6 +1,11 @@
+module "template_files" {
+  source   = "hashicorp/dir/template"
+  version  = "1.0.2"
+  base_dir = "./${var.folder_name}"
```

Comment:
I think this will work too `${path.cwd}/${var.folder_name}`

Reply:
Agree. Initially I falsely chose the `$path.module` which was not the proper approach but make sense to have this as a variable.